manifests/bosh-manifest/deployments/020-compiled-package-cache.yml

```diff
@@ -2,10 +2,7 @@ jobs:
 - name: bosh
   properties:
     compiled_package_cache:
-      provider: s3
```

Comment:
I would instead _not include this_ as it is misleading. You can set this to webdav here or local or other random string, it will be still overridden to S3. Essentially, once you specify bucket name, this property is _ignored_. Automation tools should then not parse it at all. The default is in spec, but more importantly is overridden again in erb template.